gateway/1.11/modules/ROOT/pages/policies-outbound-message-logging.adoc

```diff
@@ -0,0 +1,198 @@
+          message: "#['Upstream response: ' ++ attributes.statusCode as String]"
+          level: "INFO"
+          secondSection: true
+----
+
+include::partial$policy-title-headers.adoc[tag=ui]
+
+[%header%autowidth.spread]
+|===
+|Parameter |Description | Required?
+|Message |A DataWeave expression for extracting the message to be logged from outbound requests or upstream responses. To learn more about supported DataWeave expressions, see <<dataweave-support>>.
+|Required
+|Conditional |A DataWeave Boolean expression used to specify whether a message is to be logged. To learn more about supported DataWeave expressions, see <<dataweave-support>>. |Optional
+|Category |A prefix in the log message that indicates the package and class where the log is executed. |Optional
+|Level |The level of message to log: `INFO`, `WARN`, `ERROR`, or `DEBUG`. + 
+For more information, see <<severity-levels>>.|Required
+|Before Sending Request |Set to cause your logging policy to log messages before the outbound request is sent to the upstream service, taking into account the order in which this policy is applied. |Optional
+|After Receiving Response |Set to cause your logging policy to log messages after receiving the response from the upstream service, taking into account the order in which this policy is applied. |Optional
+|===
+
+
+[[dataweave-support]]
+== DataWeave Expressions Support
+
+The Outbound Message Logging policy supports the xref:policies-flex-dataweave-support.adoc[standard DataWeave Expressions supported by Flex Gateway policies] and the following additional xref:dataweave::dataweave-variables-context.adoc[`vars`] DataWeave expressions. Format the following expressions `#[vars.<expressions>]`, for example `#[vars.orgId]`:
+
+* `orgId`
+* `masterOrgId`
+* `envId`
+* `apiId`
+* `apiName`
+* `apiInstanceName`
+* `apiVersion`
+* `receivedTs` (rfc3339)
+* `repliedTs` (rfc3339, only available in response context)
+* `runtimeVersion` (For example: `v1.7.0`)
+* `hostId` (For example: `my-host`)
+* `reqId` (Referred to as `traceId` in Flex Gateway logs)
+* `requestDisposition` (`PROCESSED` or `VIOLATION`, only available in request context)
+* `request.remoteAddress`
+* `request.method`
+* `request.requestPath`
+* `request.headers` (all request headers)
+* `upstream.uri` (URI of the upstream service)
+* `upstream.headers` (headers sent to upstream service)
+* `upstream.statusCode` (status code from upstream response, only available in response context)
+* `violation.clientId` (Only available in response context)
+* `violation.clientName` (Only available in response context)
+* `violation.policyName` (Only available in response context)
+* `violation.violationType` (`VIOLATION` or `ERROR`, only available in response context)
+
+== How This Policy Works
+
+Logging an outbound policy consists of the following sequence:
+
+. An outbound request is prepared to be sent to an upstream service.
+. The message is logged if the following conditions are true:
```

Comment:
```suggestion
. The message is logged if these conditions are `true`:
```

Reply:
Is this "true" a variable? If not, I would rephrase to "...if these conditions are met" to avoid misunderstandings.